staging/staging.mk

```diff
@@ -0,0 +1,51 @@
+# staging.mk
+# this makefile is being included by staging/Makefile.include and staging/Makefile.dep.
+
+#
+# list of staging modules
+#
+# In order to mark a module as broken, add it to the "BROKEN" variable.
+# That way, it's dependees will be marked broken implicitly.
+#
+#STAGING_MODULES += test
+#BROKEN += test    # (broken due to change in X, see #12345)
+
+#
+# list of stating applications
+#
+# In order to mark a staging application as "broken", turn it's
+# line into a comment (and add a note).
+#
+#STAGING_APPS += broken/application/example # (intentionally broken by design)
+
+#
+# housekeeping
+#
+STAGING_MODULES := $(sort $(STAGING_MODULES))
```

Comment:
I would group all the `sort` of the variables together.

They are necessary because the `staging.mk` file will be included more than once. It could be solved by only including it once before but would currently not really add anything so it's ok to keep it this way.